client/driver/docker.go

```diff
@@ -1015,6 +1023,13 @@ func (h *DockerHandle) collectStats() {
 					s.CPUStats.CPUUsage.UsageInUsermode, s.PreCPUStats.CPUUsage.UsageInUsermode,
 					s.CPUStats.CPUUsage.TotalUsage, s.PreCPUStats.CPUUsage.TotalUsage, cores)
 
+				if h.clkSpeed == 0.0 {
```

Comment:
create `h.getClockSpeed() float64` that does this check and stores it. That way the callers are clean